app/metrics.yaml

```diff
@@ -83,3 +83,168 @@ searches:
       - dmu@mozilla.com
     expires: "2020-05-01"
+
+tabs:
+  opened:
+    type: labeled_counter
+    description: >
+      Number of opened tabs
+    send_in_pings:
+      - session_end
+    labels:
+      - context_menu
+      - tabs_dialog
+      - bookmarks
+      - history
+      - fxa_login
+      - received
+      - pre_existing
+      - browser
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1609
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  activated:
+    type: counter
+    description: >
+      Number of activated tabs
+    send_in_pings:
+      - session_end
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1609
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+firefox_account:
+  sign_in:
+    type: event
+    description: >
+      A user pressed the sign in button on the sync account page
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sign_in_success:
+    type: event
+    description: >
+      A user was successfully signed in to FxA
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sign_in_error:
+    type: event
+    description: >
+      An error happened during the login flow and the user couldn't sign in to FxA
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  sign_out:
+    type: event
+    description: >
+      A user pressed the sign out button on the sync account page and was successfully signed out of FxA
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/1610
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/2327#issuecomment-558139174
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - manmartin@mozilla.com
+    expires: "2020-05-01"
+
+  bookmarks_sync_status:
+    type: boolean
+    description: >
+      Status of the bookmarks sync engine
```

Comment:
This description is a bit unclear to me. This is a `boolean` type, how does it capture the engine status? What does `true` mean? What does `false` mean? Can you please add it to the description?

Moreover, since you're using sync, did you check that the component is not already sending what you're looking for? See [here](https://github.com/mozilla-mobile/android-components/blob/master/components/support/sync-telemetry/docs/metrics.md).

Reply:
@Dexterp37 The questions we try to answer here is:
- number of users who turn syncing off (for each bookmarks and history)

From what I see in your link, `support-sync-telemetry` doesn't answer this question

The`boolean` is the sync engine status, true for `syn on`, false for `sync off`. Applies for both supported sync engines, history and bookmarks.

Might be better for this can to use an event that we only send when the user turn the sync off?

Reply:
Oh, so this is more of a "active status" rather than an "error status". This is definitely a `boolean`. You probably want this to be a `lifetime: application` and update its value whenever the status changes, so that it's available on all the pings. See the [lifetime docs](https://mozilla.github.io/glean/book/user/adding-new-metrics.html#lifetimes).

If you want the number of users, while `events` could be checked to give you that answer, it would be more complicated. Users can toggle the status on/off in a single session, so you might receive multiple events per day. Then analyses will need to pay attention to just count *one* event per *one* user.

`boolean` with `lifetime: application` is the right choice, here. You need to clarify the description to mention what you told me above: that this is about the active/inactive state of an engine.

Reply:
Great, thanks, I'll add the application lifetime.